### DISCUSSION TITLE
Problema porte, dispositivi con HA

### DISCUSSION THREAD

**Discussion by DJTech84** [indomus/forum#3196] [Networking]
Buongiorno a tutti,
Chiedo un vostro parere perché le sto provando di ogni senza risolvere del tutto il problema.
Configurazione rete:
Iliadbox - switch - rete mesh con 3 satelliti a cui sono connessi tutti i dispositivi domotica e homeassistant. 

Sto provando ad aprire le porte del modem ma homeassistant non lo visualizzo perché è su rete mesh dopo il modem, quindi ho inserito l'IP del satellite mesh per accedere da remoto ma io da remoto non voglio visualizzare tutti i dispositivi ma solo homeassistant, come faccio a risolvere il problema?
Grazie mille a chi può aiutarmi o ci prova

**Comment by NdR91** [comment]
Non devi utilizzare l'indirizzo IP del modem, nè dei satelliti.

Devi utilizzare quello di Home Assistant, che avrai nella lista dei dispositivi.

**Reply by DJTech84** [reply]
Grazie per la risposta. Se inserisco quell'IP su locale funziona mentre da remoto con duckdns no. Il problema è che sto aprendo la porta del modem per comunicare con la casa ma o vedo i dispositivi del modem che non riguardano la domotica o i satelliti con tutti i dispositivi ma non riesco a collegarmi con home
